packages/cli/index.js

```diff
@@ -318,16 +318,38 @@ program
       .catch(handleError);
   });
 
+program
+  .command('archive [versionName] [archivePath]')
+  .alias('ar')
+  .option('-s, --site-config <file>', 'specify the site config file (default: site.json)')
+  .description('archive a version of the site, which is not affected by later changes to the site')
+  .action((versionName, userSpecifiedArchivePath, options) => {
+    if (!versionName) {
+      logger.error('Please specify a name for the archived version.');
+    }
+    const archivePath = userSpecifiedArchivePath || 'version';
+    const rootFolder = path.resolve(process.cwd());
+    const outputFolder = path.join(rootFolder, archivePath, versionName);
+    new Site(rootFolder, outputFolder, undefined, undefined, options.siteConfig)
+      .archive(versionName, archivePath)
+      .then(() => {
+        logger.info(`Successfully archived ${versionName} at ${archivePath}/${versionName}`);
+      })
+      .catch(handleError);
+  });
+
 program
   .command('deploy')
   .alias('d')
   .description('deploy the site to the repo\'s Github pages')
   .option('-c, --ci [githubTokenName]', 'deploy the site in CI Environment [GITHUB_TOKEN]')
   .option('-s, --site-config <file>', 'specify the site config file (default: site.json)')
+  .option('-v, --versions <versionFolder>', 'specify the path where versions are stored')
```

Comment:
I suppose this is not documented yet? Not sure if it is ready for testing.

Reply:
I added this originally when I thought it might be good to specify exactly which versions should be deployed, but now I think this should be left to a later PR to think about implementation! For now I think moving over all archived versions (or manually excluding in site.json) is the most natural solution and least prone to error.